client-android/src/main/java/no/nordicsemi/android/kotlin/ble/client/real/NativeClientBleAPI.kt

```diff
@@ -84,55 +86,55 @@ class NativeClientBleAPI(
         value: DataByteArray,
         writeType: BleWriteType
     ) {
-        val characteristic = (characteristic as NativeBluetoothGattCharacteristic).characteristic
+        val c = (characteristic as NativeBluetoothGattCharacteristic).characteristic
```

Comment:
Why such a short name? If the code is long enough you will have to look what this variable means.

Reply:
There was a warning, that the new `characteristic` hides the method parameter.
I agree, that `c` is a bad name. I'l revert.